pkgs/microcap/default.nix

```diff
@@ -37,11 +37,16 @@ in mkWindowsApp rec {
     # This file is changed on runtime, so we'll have to link it
     "$HOME/.config/Micro-Cap/mcap.dat" = "drive_c/MC12/MCAP.dat";
```

Comment:
File mappings are case sensitive. If I recall, the file name needs be lowercase: drive_c/MC12/mcap.dat